deployments/ansible/roles/signalfx-agent/tasks/yum_repo.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Add Signalfx Agent repo into source list
+  yum_repository:
+    name: signalfx-agent
+    description: SignalFx Agent Repository
+    baseurl: "{{ rhel_repo_url }}"
+    gpgkey: "{{ rhel_gpg_key_url }}"
+    gpgcheck: yes
+    enabled: yes
+
+- name: Install Signalfx Agent via yum package manager
+  yum:
+    name: signalfx-agent
+    state: present
+
+- name: add signalfx group
```

Comment:
You don't really need to add the user/group after the package is installed, since the install scripts in the package do that for both Debian and RHEL.  It only needs to be done before the package is installed if you want to render the config file with the right owner/group, but you can just do it after the package is installed so that it is never necessary to make a user/group.

Reply:
got it, removed the tasks to add user/group explicitly